pyproject.toml

```diff
@@ -27,6 +27,9 @@ pydantic = ">=1.10.15"
 cryptography = ">=1.9"
 async-timeout = ">=3.0.0"
 aiohttp = ">=3"
+# temporary pin for indirect cffi dependency (req of cryptography)
+# to be removed pre-merge, just here for testing, see https://github.com/python-cffi/cffi/issues/71
+cffi = { version = ">=1.17.0rc1", python = ">=3.13" }
```

Comment:
https://github.com/python-cffi/cffi/releases/tag/v1.17.0 is now out and available on pypi